src/components/ClusterTemplateDetails/QuotasSection.tsx

```diff
@@ -40,19 +41,13 @@ const _QuotasSection: React.FC<{ quotas: ClusterTemplateQuota[] }> = ({ quotas }
         />
       </StackItem>
       <StackItem>
-        <ClusterTemplateQuotasTable quotas={quotas} />
+        <TableLoader loaded={loaded} error={error}>
+          {quotasDetails.length === 0 && <QuotasEmptyState />}
+          {quotasDetails.length > 0 && <ClusterTemplateQuotasTable quotaDetails={quotasDetails} />}
```

Comment:
we could reduce this to just a ternary operator 
```suggestion
          {quotasDetails.length === 0 ? <QuotasEmptyState /> : <ClusterTemplateQuotasTable quotaDetails={quotasDetails} />}
```